src/server-middleware/jira-service-desk-api.js

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+
+const encodedAuthString = Buffer.from(`${process.env.JIRA_API_USER}:${process.env.JIRA_API_TOKEN}`).toString('base64');
+
+export default {
+  path: '/jira-servicedesk',
+  handler(req, res) {
+    if (req.method === 'POST') {
+      let data = '';
+      req.on('data', chunk => {
+        data += chunk;
+      }).on('end', () => {
+        let parsedData = (JSON.parse(data));
+        const createRequestData = {
+          serviceDeskId: '7',
+          requestTypeId: '81', // Type 81 is 'suggestion'
+          requestFieldValues: {
+            summary: parsedData.summary
+          }
+        };
+        if (parsedData.email) {
+          createRequestData.raiseOnBehalfOf = parsedData.email;
+        }
+
+        axios.post('https://europeana.atlassian.net/rest/servicedeskapi/request', createRequestData, {
```

Comment:
Should the endpoint could be passed via env var?

Reply:
Yes, and is in my refactor.